lib/configuration_methods.rb

```diff
@@ -69,6 +69,25 @@ def secret_token
       end
     end
 
+    def twofa_encryption_key
+      if heroku?
+        return ENV['TWOFA_ENCRYPTION_KEY'] if ENV['TWOFA_ENCRYPTION_KEY']
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.